src/components/main-form.js

```diff
@@ -519,19 +546,93 @@ class MainForm extends React.Component {
                   and your return on investment lower. We factor all of this
                   automatically so the final result you see on your right is not
                   the actual money you will have gained/lost in{' '}
-                  {this.state.longerOption.term} years, but rather, the money
-                  you will have gained/lost in {this.state.longerOption.term}{' '}
-                  years adjusted to today’s purchasing power. If you’re curious
-                  about what the actual number would be, give inflation a value
-                  of 0%. But remember that everything will be a lot more
-                  expensive in {this.state.longerOption.term} years, and that is
-                  why we need to include inflation.
+                  {longerOption.term} years, but rather, the money you will have
+                  gained/lost in {longerOption.term} years adjusted to today’s
+                  purchasing power. If you’re curious about what the actual
+                  number would be, give inflation a value of 0%. But remember
+                  that everything will be a lot more expensive in{' '}
+                  {longerOption.term} years, and that is why we need to include
+                  inflation.
                 </Accordion>
               </FormGroup>
             </Form>
           </Card>
         </Col>
         <Col xs='8'>
+          <Summary className='p-4 mr-5'>
+            <h4>Explanation</h4>
+            <p>
+              <b>{formatMoney(this.state.optCost)}</b> is the difference between
+              your {shorterOption.term} year investment total (
+              <b>{formatMoney(shorterOption.fv)}</b>) and your{' '}
+              {longerOption.term} year investment total (
+              <b>{formatMoney(longerOption.fv)}</b>).
+            </p>
+            <Col>
+              <h5>{shorterOption.term} year scenario:</h5>
+              <ol>
+                <li>
+                  With the <b>{shorterOption.term} year mortgage</b> you pay{' '}
+                  <b>{formatMoney(shorterOption.pmt)}</b> monthly for{' '}
+                  {shorterOption.term} years.
+                </li>
+                <li>
+                  After the house is paid, you{' '}
+                  <b>invest {formatMoney(shorterOption.pmt)}</b> monthly with an{' '}
+                  <b>ROI of {this.state.investmentRate}%</b> at an{' '}
+                  <b>inflation rate of {this.state.inflation}%</b>.
+                </li>
+                <li>
+                  After <b>{longerOption.term} years</b>, your{' '}
+                  <b>investments</b> are worth{' '}
+                  <b>{formatMoney(shorterOption.fv)}</b>.
+                </li>
+              </ol>
+            </Col>
+            <span className='border mr-5' />
```

Comment:
sug: You could probably just use a `border-right` on the prior column

Reply:
it might be harder to center it that way, did you give it a try?